HarrisMoviesApplication/ViewController.swift

```diff
@@ -8,13 +8,42 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    struct CatFact: Decodable {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
-        //creating
-    }
+        // MARK: - Properties
 
+        let text: String
+    }
+    
+    let factsArray:[CatFact] = []
 
+    @IBOutlet weak var catFactTableView: UITableView!
+    
+    override func viewDidLoad() {
```

Comment:
```suggestion
    override func viewDidLoad() {
    super.viewDidLoad()
```

Reply:
i already move it, you can see it in the last commit